tests/RestfulDbQueryTestCase.test

```diff
@@ -68,19 +65,20 @@ class RestfulDbQueryTestCase extends DrupalWebTestCase {
       'integer' => $mock_data['int_field'],
       'string' => $this->randomName(),
     );
+
     $handler->put($id, $mock_data2);
     $result = $handler->get($id);
-    // Remove nullified fields.
-    $result = array_filter($result[0]);
-    $this->assertEqual($result, $mock_data2, 'The record was updated with PUT successfully.');
+    // Make mock data resemble the intended response.
+    $mock_data2['serialized'] = $mock_data['serialized_field'];
```

Comment:
Actually this is the same `PUT` and `PATCH` situation as before. If we do not provide a _serialized_ and we call `PUT` that means that the _serialized_ should be nullified.